docs/en/enterprise-edition/content-collections/connect/connect-cloud-accounts/onboard-your-azure-account/authorize-prisma-cloud.adoc

```diff
@@ -277,19 +277,19 @@ image::connect/azure-account-view-roles.png[]
 .. Navigate to Microsoft Graph. Select menu:API{sp}permissions[+ Add a permission > Microsoft Graph > Application permissions].
 .. Add the permissions. Enter the permission name in *Select permissions*, and select the name from *Permission*. Add the following permissions:
 
-* screen:[User.Read.All]
-* screen:[Policy.Read.All]
-* screen:[Group.Read.All]
-* screen:[GroupMember.Read.All]
-* screen:[Reports.Read.All]
-* screen:[Directory.Read.All]
-* screen:[Domain.Read.All]
-* screen:[Application.Read.All]
+* User.Read.All
```

Comment:
the screen:[User.Read.All] syntax must be replaced with `User.Read.All`.

Reply:
Done